tgstation.dme

```diff
@@ -3639,6 +3639,7 @@
 #include "fulp_modules\maps\fulp_shuttles\fulp_shuttles.dm"
 #include "fulp_modules\ZFulpcode\modules\jobs\huds.dm"
 #include "fulp_modules\ZFulpcode\modules\jobs\inventories.dm"
+#include "fulp_modules\edits\item_desc\item_desc_edit.dm"
```

Comment:
.dme files are ordered alphabetically, this will make it so every following PR moves this file to where it's supposed to be due to it being re-ordered automatically

Reply:
Simply updated the PR with the auto organized dme file, moving it to the correct place, I hope